src/client/components/DeliveryInfo/DeliveryInfo.styles.css

```diff
@@ -0,0 +1,168 @@
+form {
+  display: grid;
+  padding: 1em;
+  background: #f9f9f9;
+  border: 1px solid #c1c1c1;
+  margin: 1rem auto 0 auto;
+  max-width: 600px;
+  padding: 1em;
+}
+form input {
+  background: #fff;
+  border: 1px solid #9c9c9c;
+}
+form button {
+  background: #8e0ef2;
+  padding: 0.7em;
+  width: 100%;
+  border: 0;
+}
+form button:hover {
+  background: grey;
+}
+
+label {
+  padding: 0.5em 0.5em 0.5em 0;
+}
+
+input {
+  padding: 0.7em;
+  margin-bottom: 0.5rem;
+}
+input:focus {
+  outline: 3px solid #8e0ef2;
+}
+
+@media (min-width: 400px) {
```

Comment:
Looks like there are a lot of stuff in this media query. I can't go into detail now, but try to look if you can avoid some replication of CSS.